bot/cogs/utility/report.py

```diff
@@ -0,0 +1,125 @@
+            embed.add_field(name="Reported Message Content", value=f"{reported_msg.content}")
+        else:
+            embed.add_field(name="Reported Message Content (1)", value=f"{reported_msg.content[:1024]}")
+            embed.add_field(name="Reported Message Content (2)", value=f"{reported_msg.content[1025:]}")
+        embed.add_field(name="Reported by:", value=f"{ctx.author.mention}")
+        embed.add_field(name="Link to message:", value=f"{reported_msg.jump_url}")
+        await ctx.message.add_reaction("✅")
+        message = await acc_channel.send(embed=embed)
+        await message.add_reaction("✅")
+        await message.add_reaction("❌")
+        await asyncio.sleep(5)
+        await ctx.message.delete()
+
+    @commands.Cog.listener(name="on_raw_reaction_add")
+    async def on_raw_reaction_add(self, payload):
+        # this rules out other reactions
+        if payload.channel_id != accept_channel_id or payload.user_id == self.bot.user.id:
+            return
+        emoji = payload.emoji.name
+        if emoji not in ['✅', '❌']:
+            return
+
+        channel = self.bot.get_channel(payload.channel_id)
+        msg = await channel.fetch_message(payload.message_id)
+        easy_embed = msg.embeds[0].to_dict()
+
+        server = self.bot.get_guild(msg.guild.id)
+        reactor = server.get_member(int(payload.user_id))
+
+        field_reported_msg_author = easy_embed['fields'][0]['value']
+        if easy_embed['fields'][1]['name'] == "Reported Message Content":
+            field_reported_msg_content = easy_embed['fields'][1]['value']
+        else:
+            field_reported_msg_content_1 = easy_embed['fields'][1]['value']
+            field_reported_msg_content_2 = easy_embed['fields'][2]['value']
+        field_reporter = easy_embed['fields'][-2]['value']
+        field_url = easy_embed['fields'][-1]['value']
+
+        rp_channel = server.get_channel(int(field_url[-37:-19]))
+        try:
+            rp_message = await rp_channel.fetch_message(int(field_url[-18:]))
+        except discord.NotFound:
+            embed = discord.Embed(title="Reported Message:", color=15105570)
+            embed.set_author(name="Warning: Message couldn't be found")
+            embed.add_field(name="Reported Message Author", value=f"{field_reported_msg_author}")
+            if field_reported_msg_content:
+                embed.add_field(name="Reported Message Content", value=f"{field_reported_msg_content}")
+            else:
+                embed.add_field(name="Reported Message Content (1)", value=f"{field_reported_msg_content_1}")
+                embed.add_field(name="Reported Message Content (2)", value=f"{field_reported_msg_content_2}")
+            embed.add_field(name="Reported by:", value=f"{field_reporter}")
+            embed.add_field(name="Link to message:", value=f"{field_url}")
+            await msg.edit(embed=embed)
+            await msg.clear_reactions()
+            pass
+            return
```

Comment:
`pass` and `return`???

Reply:
Yes, as I want the function to end if an exception occurs as well as avoid clogging up the log.

Reply:
It should just be `return`, `pass` is a no-op statement

Reply:
🤦 I guess i'm way too tired